tests/Test.ADAL.NET.Unit/OboFlowTests.cs

```diff
@@ -110,7 +110,8 @@ public void ResetInstanceDiscovery()
                 PostData = new Dictionary<string, string>()
                 {
                     {"client_id", TestConstants.DefaultClientId},
-                    {"grant_type", "urn:ietf:params:oauth:grant-type:jwt-bearer"}
+                    {"grant_type", "urn:ietf:params:oauth:grant-type:jwt-bearer"},
+                    {OAuthParameter.ClientInfo, "1"}
```

Comment:
>"1" [](start = 48, length = 3)

pls. consider making this a meaningfully named constant as that serves as good documentation as to why this is set to "1" (e.g. REQUEST_CLIENT_INFO or something similar)